src/Collector/Stream/HttpStreamCollector.php

```diff
@@ -76,9 +90,7 @@ public function getSummary(): array
             'http_stream' => array_merge(
                 ...array_map(
                     fn (string $operation) => [
-                        $operation => is_countable($this->requests[$operation]) ? count(
-                            $this->requests[$operation]
-                        ) : 0,
+                        $operation => count($this->requests[$operation]),
```

Comment:
I don't remember why, but it was not just for fun

Reply:
`requests` property always contains array of lists (type `array<string, list<array{uri: string, args: array}>>`)
So countable check is not need.

Reply:
Maybe it was originally in the http stream proxy and so the requests were overridden in php engine, but anyway let's pass it now, will check it later